### ISSUE TITLE
Django TemplateSyntaxError in smtweb

### ISSUE THREAD

**Issue by maxalbert** [open-research/sumatra#254]
When running smtweb in my Sumatra project I get the following django error in the browser:

```
TemplateSyntaxError at /
'filters' is not a valid tag library: Template library filters not found, tried
django.templatetags.filters,tagging.templatetags.filters
```

This is with a Python 3.4.3, Django 1.8.2 and latest Sumatra master (55e3abc8769f773c51). Any ideas what could be causing it? Please let me know if you need any further debugging information.

**Comment by apdavison** [comment]
I think I've fixed this in 0a62d9a. Please can you check it now works for you, and close this ticket if everything's ok.

**Comment by maxalbert** [comment]
Yep, that fixed it. :) Thanks!
